tests/e2e/overpaying.spec.ts

```diff
@@ -0,0 +1,237 @@
+import { spy } from 'tinyspy';
+import { test, expect } from './fixtures/connected';
+import {
+  getLastCallArg,
+  playgroundUrl,
+  setupPlayground,
+} from './helpers/common';
+import { sendOneTimePayment } from './pages/popup';
+import type { OutgoingPayment } from '@interledger/open-payments';
+
+test.beforeEach(async ({ popup }) => {
+  await popup.reload();
+});
+
+test.afterEach(({ persistentContext: context }) => {
+  context.removeAllListeners('requestfinished');
+});
+
+const walletAddressUrl = process.env.TEST_WALLET_ADDRESS_URL;
+
+test.describe('should not pay immediately when overpaying', () => {
```

Comment:
Spent the last couple of minutes looking for a test that goes through this flow without realizing that I already went through it. PEBCAK 🙄